testing/define_testing_alias.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
I don't see the purpose of this. It doesn't save the user any lines or any characters.

```bash
./testing/define_testing_alias.sh
tfp_test
```

vs

```
./testing/tfp_test.sh
```

It also feels too presumptive of people's bash habits.

Reply:
Well, it's not required to use. But if you're cd'ed several dirs deep into the code then you need a bunch of ../'s to find the testing dir and the script. i'm assuming one would define the alias once and then use it a bunch of times in a session, so in the long run it'd save many typed chars. If this is strongly opposed I can remove it but I'd certainly like to have it around myself.

Reply:
ie, with this i can do

```
source ./testing/define_testing_alias.sh
cd tensorflow_probability/python/mcmc
tfp_test :hmc_test
# ...
tfp_test :dual_averaging_step_size_test
# etc
```

vs

```
cd tensorflow_probability/python/mcmc
../../../testing/tfp_test.sh :hmc_test
# ...
../../../testing/tfp_test.sh :dual_averaging_step_size_test
# etc
```

Reply:
Just don't cd into the subdirectory and liberally use bash history. Like I said, different people have different habits. I guess it's fine to leave this here, the verbiage in the readme is non-committal enough to not make it seem that this is mandatory.

Reply:
Thanks, Pavel. +@ColCarroll to get another perspective on whether it's weird to have this around. I don't feel super strongly, but it would be convenient for my usual workflow/habits.